docs.feldera.com/docs/pipelines/modifying.md

```diff
@@ -0,0 +1,436 @@
+# Modifying a Pipeline
+
+:::note Enterprise-only feature
+This feature is only available in Feldera Enterprise Edition.
+:::
+
+## Overview
+
+Feldera supports **incremental pipeline modification** through a feature called **bootstrapping**.
+This allows you to evolve your SQL pipelines by adding, removing, or modifying tables and views
+without having to re-ingest all historical data from scratch.
+
+When you modify a pipeline:
+- **New and modified views** are computed from existing pipeline state
+- **Unchanged views** continue using their existing state
+- **Input data** does not need to be re-ingested (unless table schemas change)
+
+## Introduction
+
+Feldera incrementally processes changes to input data. But what happens when the program
+itself needs to change, e.g., the user would like to add, remove, or modify a view definition?
+Feldera is able to handle such changes incrementally as well by only recomputing
+the modified view and any views derived from it. It uses existing state in tables
+and internal indexes maintained by the incremental view maintenance engine
+as input to the computation.
```

Comment:
this last line I don't understand. What "computation" is referred here?
I would say that the system attempts to reuse as much state as possible from the previous execution of the pipeline, assuming the user intent is to continue executing the existing pipeline based on the whole data ingested so far